src/EventHandler.tsx

```diff
@@ -223,7 +223,9 @@ export class EventHandler extends React.PureComponent<EventsHandlerProps> {
    */
   handleMouseEvent = (e: React.MouseEvent<HTMLDivElement>) => {
     const { handleMouseEvent } = this.props;
-
+    if (e.shiftKey) {
+      this.handleCopy();
```

Comment:
I'm confused by this bit, why does it copy every time the cursor moves when shift selecting? I feel like I'd never expect that + doesn't cmd + c work just as well? I also find it surprising to click a single amino acid but have the entire translation copied

Reply:
Yea this wasn't the best implementation for copying the highlighted aa sequence. For some reason a normal  cmd + c  doesn't work with a highlighted aa sequence (selecting using the click + shift method, not drag select). I tried a work around but for now I'm going to remove that